src/EnergyPlus/General.cc

```diff
@@ -3922,6 +3922,23 @@ namespace General {
         return results;
     }
 
+    Real64 epexp(Real64 x)
+    {
+        if (x < -70.0) {
+            return 0.0;
+        }
+        return std::exp(x);
+    }
+
+    Real64 epexp(Real64 x, Real64 defaultHigh)
+    {
+        if (x < -708.4) {
```

Comment:
@lgu1234 I'm confused here. Why does this form use -708.4 as the lower limit while the one above uses -70.0? Also, the unit test does not exercise this form.

Reply:
If I recall correctly, we discussed having one function with both high and low tests. So this second one is more for completeness and potential use outside AFN. Could also do it like this:

```c++
    Real64 epexp(Real64 x, Real64 defaultLow=-70.0)
    {
        if (x < defaultLow) {
            return 0.0;
        }
        return std::exp(x);
    }

    Real64 epexp(Real64 x, Real64 defaultHigh, Real64 defaultLow=-70.0)
    {
        if (x < -defaultLow) {
            return 0.0;
        } else if (x > defaultHigh) {
            return std::exp(defaultHigh);
        }
        return std::exp(x);
    }
```

But it probably should be the same if we leave it as is.

Reply:
@mjwitte This is my mistake. The value should be -70.0 in both functions. In addition, I will add a unit test for the second function.

@jasondegraw I like your suggestion. However, I got compile problem, because polymorphism may not allow different types of arguments: overloaded in the first equation, and normal in the second equation. Then, I changed both equations slightly with argument order:
    Real64 epexp(Real64 x, Real64 defaultLow=-70.0)
    {
        if (x < defaultLow) {
            return 0.0;
        }
        return std::exp(x);
    }

    Real64 epexp(Real64 x, Real64 defaultLow=-70.0, Real64 defaultHigh = 700.0)
    {
        if (x < defaultLow) {
            return 0.0;
        } else if (x > defaultHigh) {
            return std::exp(defaultHigh);
        }
        return std::exp(x);
    }
Visual Studio still gives me an error.

Since we agree -70 for low end, I am going to hard-wired the value in both functions:

    Real64 epexp(Real64 x)
        if (x < -70.0) {
            return 0.0;
        }
        return std::exp(x);
    }

    Real64 epexp(Real64 x, Real64 defaultHigh)
    {
        if (x < -70) {
            return 0.0;
        } else if (x > defaultHigh) {
            return std::exp(defaultHigh);
        }
        return std::exp(x);
    }